src/components/project/WithHeaderHoverCard.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+export type BlockProps = {
+  header: string;
+};
+
+const WithHeaderHoverCard: React.FC<BlockProps> = (props) => {
+  const { header } = props;
+  return (
+    <div className="pc:w-60 pc:h-60 desktop:w-96 desktop:h-96 bg-greenC7D1BD rounded-2xl shadow-2xl transition duration-500 ease-in-out transform hover:scale-110 flex justify-center items-center">
+      <div className="text-2xl">{header}</div>
```

Comment:
![image](https://user-images.githubusercontent.com/34268371/137584860-9884abbd-19d6-49a7-ad3a-1d434d81485c.png)

ホバーカードのサイズが小さくて文字寄せが左になってしまうので、 `text-center` をつけた方がいいと思います。

また、横に padding があった方が綺麗だと思うので、`px-2` や `px-1` をつけた方がいいと思います。

```suggestion
      <div className="text-2xl text-center px-2">{header}</div>
```